snapshots.go

```diff
@@ -390,6 +394,14 @@ func (s *Stream) createSnapshot(ctx context.Context, dataBuffer, metadataBuffer
 	}
 
 	sub, err := s.mgr.nc.Subscribe(ib, func(m *nats.Msg) {
+		clientInfoHeader := m.Header.Get(server.ClientInfoHdr)
+
+		// if the server returns a non-204 status code in the message header, return an error
+		if !strings.Contains(clientInfoHeader, strconv.Itoa(http.StatusNoContent)) {
```

Comment:
minor nit but if you're making a constant for above lets also do one for this here so we dont keep caling Itoa

Reply:
I'll just use a string for the status code